docs/guides/darwin.md

```diff
@@ -0,0 +1,179 @@
+   instructions
+   [above](#ensuring-your-matter-accessory-works-with-iosipadostvos)
+
+2. Edit PRODUCT_ID value in the file
+   `connectedhomeip/examples/lighting-app/nrfconnect/main/include/CHIPProjectConfig.h`
+   to ensure you are using a supported Product ID from the “Ensuring Your Matter
+   Accessory Works with iOS iPadOS/tvOS” section above.
+    - Follow
+      [these instructions](https://github.com/project-chip/connectedhomeip/tree/master/examples/lighting-app/nrfconnect#readme)
+      initialize your development environment, compile the firmware and flash
+      your hardware.
+
+##### Configuring Multiple Hardware Accessories
+
+1. Follow instructions 1 through 5 above for your 2nd of accessory hardware
+2. On your hardware, Navigate to "Component Config" → p device layer → Testing
+   Options.
+3. Edit both the "Use Test Setup Pin Code" and "Use t Setup discriminator" to be
+   unique values not shared with other Hardware. For example: change the last
+   digit of each.
+
+##### Configuring other Platforms
+
+##### Note: Most platforms have very similar, if not the same configuration requirements
+
+1. Checkout and setup
+   [Matter repo](https://github.com/project-chip/connectedhomeip.git) as per the
+   instructions
+   [above](#ensuring-your-matter-accessory-works-with-iosipadostvos)
+2. Find and edit one of the platform
+   [examples](https://github.com/project-chip/connectedhomeip/tree/master/examples)
+   to support the fixed device types
+   [above](#ensuring-your-matter-accessory-works-with-iosipadostvos)
+3. Read the [platform guides](.) on how set up the hardware
+
+##### Example guides as of editing this page
+
+-   [Bouffalo Lab](/examples/lighting-app/bouffalolab/bl602/README.md)
+-   [EFR32 Window Covering](/examples/window-app/efr32/README.md)
+-   [ESP32 All Clusters](/examples/all-clusters-app/esp32/README.md)
+-   [ESP32 Lighting](/examples/lighting-app/esp32/README.md)
+-   [ESP32 Temperature Sensor](/examples/temperature-measurement-app/esp32/README.md)
+-   [mbedOS](/examples/all-clusters-app/mbed/README.md)
+-   [nRF Connect All Clusters](./nrfconnect_examples_configuration.md)
+-   [nRF Connect Pump](/examples/pump-app/nrfconnect/README.md)
+-   [NXP Examples](./nxp_imx8m_linux_examples.md)
+-   [NXP](/examples/all-clusters-app/nxp/mw320/README.md)
+-   [P6](/examples/all-clusters-app/p6/README.md)
+-   [Qorvo](/examples/lighting-app/qpg/README.md)
+-   [SiliconLabs](./silabs_efr32_building.md)
+-   [Simulated Linux](./simulated_device_linux.md)
+-   [Telink](/examples/lighting-app/telink/README.md)
+-   [TI Platform](./ti_platform_overview.md)
+-   [TI All Clusters](/examples/all-clusters-app/cc13x2x7_26x2x7/README.md)
+-   [Tizen](/examples/lighting-app/tizen/README.md)
+
+### Release Notes & Known Issues
+
+-   Please refer to the iOS/iPadOS 15.6
+    [Release Notes](https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-15_6-release-notes)
```

Comment:
Please also add release notes for iOS 16.0 at https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-16-release-notes.

Reply:
Done, ty